code/clue.js

```diff
@@ -40,20 +224,45 @@ const randomSelector = array => {
 // CREATE AN OBJECT THAT KEEPS THE MYSTERY.
 // With a killer, a weapon and a room.
 // The values will be set later.
+const mystery = {
+  killer: '',
+  weapon: '',
+  room: '',
+}
+
+// PICK KILLER when killerCard is clicked.
 
-// This function will be invoked when you click on the killer card.
 const pickKiller = () => {
-  // This will randomly select a killer from the suspects. And add that to the mystery object.
   mystery.killer = randomSelector(suspects)
-
-  // This will change the background color of the card to the one connected to the chosen killer and show the full name of the killer. Feel free to add more things to show about the killer.
   document.getElementById('killerCard').style.background = mystery.killer.color
-  document.getElementById(
-    'killerName'
-  ).innerHTML = `${mystery.killer.firstName} ${mystery.killer.lastName}`
+  document.getElementById('killerName').innerHTML = `${mystery.killer.firstName} ${mystery.killer.lastName}`
+  document.getElementById('killerImage').src = mystery.killer.image
+  document.getElementById('killerOccupation').innerHTML = mystery.killer.occupation
+  document.getElementById('killerDescription').innerHTML = mystery.killer.description
+}
+document.getElementById('killerCard').addEventListener('click', pickKiller)
+
+// PICK WEAPON when weaponCard is clicked.
+
+const pickWeapon = () => {
+  mystery.weapon = randomSelector(weapons)
+  document.getElementById('weaponName').innerHTML = mystery.weapon.name
+  document.getElementById('weaponMaterial').innerHTML = mystery.weapon.material
 }
+document.getElementById('weaponCard').addEventListener('click', pickWeapon)
 
-// CREATE FUNCTIONS pickWeapon and pickRoom in a similar way.
+// PICK ROOM when roomCard is clicked.
+
+const pickRoom = () => {
+  mystery.room = randomSelector(rooms)
+  document.getElementById('roomName').innerHTML = mystery.room.name
+}
+document.getElementById('roomCard').addEventListener('click', pickRoom)
 
 // STEP 4 - CREATE A FUNCTION revealMystery that will be invoked when you click that button. It should show something like:
-// 'The murder was committed by Jacob Green, in the living room with a rope.'
+// 'The murder was committed by Jacob Green, in the living room with a rope. */
```

Comment:
Not always!
```suggestion
```

Reply:
I don''t understand this comment. This line was already in the files, it's not a part of my code.

Reply:
It's a comments that's there for helping you when you start out this project. But now when you are done with this comment doesn't make sence anymore so you can delete it from your project